src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionCoordinator.swift

```diff
@@ -258,6 +258,7 @@ class ExposureSubmissionCoordinator: NSObject, RequiresAppDependencies {
 				guard let self = self, let coronaTest = self.model.coronaTest else { return }
 
 				guard coronaTest.isSubmissionConsentGiven else {
+					self.model.createDiaryTestEntry()
 					self.showTestResultScreen()
```

Comment:
Would it make sense to move the call to self.showTestResultScreen?

Reply:
the showTestResultScreen function is creating a screen and pushing it i thought it will be irrelevant to the function, but i moved it